cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -184,6 +184,34 @@ spec:
       for: 1h
       labels:
         severity: warning
+  - name: persistent-volume-alert.rules
```

Comment:
To be consistent with existing alerts (KubePersistentVolumeUsageCritical; only for openshift-* namespace) I wonder if using these alert names would be better:
PersistentVolumeUsageNearFull
PersistentVolumeUsageCritical

Reply:
Ack, Will update.